### PR TITLE
Update button.ts

### PR DESCRIPTION
**Pull request by aWeirdo**
Added static "CreateImageWithCenterTextButton" with centered text and stretching image background.
Which i believe is a very common button setup.

https://www.babylonjs-playground.com/#XCPP9Y#288

i wasn't sure what to name it.. i think it ended up rather long, any suggestions?

### CODE REVIEW

File: gui/src/controls/button.ts [hunk 1]
```diff
@@ -136,5 +136,23 @@ module BABYLON.GUI {
...
+            // Adding text
+            var textBlock = new BABYLON.GUI.TextBlock(name + "_button", text);
+            textBlock.textWrapping = true;
+            textBlock.zIndex = 10; 
+            textBlock.textHorizontalAlignment = BABYLON.GUI.Control.HORIZONTAL_ALIGNMENT_CENTER;
+            result.addControl(textBlock);   
+
+            // Adding image
+            var iconImage = new BABYLON.GUI.Image(name + "_icon", imageUrl);
+            iconImage.stretch = BABYLON.GUI.Image.STRETCH_FILL;
```
Comment (deltakosh) [inline gui/src/controls/button.ts]: Should the image be behind the text?

Comment (aWeirdo) [reply gui/src/controls/button.ts]: Correct, the image is a background.
Here's an example which isn't grass :)
https://raw.githubusercontent.com/aWeirdo/Babylon.js/master/button.png

Comment (deltakosh) [reply gui/src/controls/button.ts]: So you should add the image first

Comment (aWeirdo) [reply gui/src/controls/button.ts]: i'll swap them around if you prefer.
i just did it with zIndex as i liked keeping the textPanel as the first child.

Comment (deltakosh) [reply gui/src/controls/button.ts]: The zindex will reord the children

Comment (aWeirdo) [reply gui/src/controls/button.ts]: Ahh, i see, when the control is added it..

**Pull request merged**